services/soundplayer.md

```diff
@@ -0,0 +1,38 @@
+# Sound player
+
+A device that can play various sounds stored locally. This service is typically paired with a ``storage`` service for storing sounds.
+
+    identifier: 0x1e3048f8
+
+## Registers
+
+    rw volume: u16 @ intensity
+
+Global volume of the output. ``0`` means completely off. This volume is mixed with each play volumes.
+
+    const supported_wav_format? @ 0x80 {
+        stereo: bool
+        bit: u8
+        max_sample_rate: u16 Hz
```

Comment:
I think this should u32, first field for alignment. There may be situations where you want more than 65kHz